check_translation.php

```diff
@@ -0,0 +1,595 @@
+#!/usr/bin/php
```

Comment:
let's move these helper files inside the `scripts/helpers` directory.

Reply:
`scripts` if for kalkun scripts, not for devel or helper scripts.
We can add them in another toplevel dir, but I prefer not into `scripts`.

Reply:
That also works, please create one. Let's not keep these in root dir.

Reply:
Any suggestion for a short name?

Reply:
I think `utils` and `docs` would be fine with files placed in it as suggested in my previous comment. Are you ok with that?

Reply:
Sounds good to me 👍

Reply:
Done for this script.
Other are in https://github.com/kalkun-sms/Kalkun/pull/386
Please review again.